From 3ec4d99d8f853fe68c39e4d389120bd96861cdb0 Mon Sep 17 00:00:00 2001
From: Davidson K <davidson.kumaresan@arm.com>
Date: Thu, 1 Dec 2022 19:33:34 +0530
Subject: [PATCH 2/2] feat(plat/tc): change the properties of optee reserved
 memory

Removing optee node (or) removing "no-map" will help kernel can create
Stage-1 VA mappings to this region.  So, It will be good if we make this
region as  "restricted-dma-pool". The use of the restricted dma pool is
memory meant to be used for a set devices and region is accessible to only
for those devices .

In this case, there is no device using it from linux , it will be remained
restricted and untouched.

Signed-off-by: Davidson K <davidson.kumaresan@arm.com>
Change-Id: Id8e8c7034c3f2e5e0414bc74ca5ade3aac5e2e7d
Upstream-Status: Pending [Not submitted to upstream yet]
---
 fdts/tc.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fdts/tc.dts b/fdts/tc.dts
index 101b11a60..067c0f4ec 100644
--- a/fdts/tc.dts
+++ b/fdts/tc.dts
@@ -214,8 +214,8 @@
 		};
 
 		optee@0xf8e00000 {
+			compatible = "restricted-dma-pool";
 			reg = <0x00000000 0xf8e00000 0 0x00200000>;
-			no-map;
 		};
 
 		fwu_mm@0xfca00000 {
-- 
2.34.1

